actnow/petitions/models.py

```diff
@@ -0,0 +1,71 @@
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+from actnow.accounts.models import ActNowUser
+from actnow.db.models import TimestampedModelMixin
+
+
+class Petition(TimestampedModelMixin):
+    title = models.CharField(
+        _("title"),
+        max_length=255,
+    )
+    description = models.CharField(
+        _("details of petition"),
+        max_length=1024,
```

Comment:
From [this](https://docs.djangoproject.com/en/3.2/ref/databases/#character-fields), I think it would be better to use a [TextField](https://docs.djangoproject.com/en/3.2/ref/databases/#character-fields)